docs/content/documentation/flink-table-store-0.3.md

```diff
@@ -1,7 +1,7 @@
 ---
-weight: 10
-title: Table Store Master (snapshot)
-bookHref: "https://nightlies.apache.org/flink/flink-table-store-docs-master/"
```

Comment:
We can just remove this file.

Reply:
I prefer to keep this file as flink-table-store was part of history of Flink, and developers/users could still find the valid legacy links if necessary.

Reply:
Why? With Paimon now becoming its own project, I don't think we should per se link to it from the Flink project website. 

The history of Flink can always be found in Git anyway.

Reply:
I think we should still host the docs of legacy version of Paimon for users, and it seems we cannot find the legacy docs of older version via Paimon's website. That's why I prefer to keep it.

Reply:
I understand that, but I don't think we should link to the legacy docs from the main menu. It will just redirect users to outdated content, while they should be checking out the latest docs. 

Paimon could consider linking to the old docs from their project website. Another approach would be that the announcement of Paimon itself would be posted as a Flink blog, and that includes links to the old doc versions there.